newsroom/section_filters/model.py

```diff
@@ -0,0 +1,12 @@
+from typing import Optional
+from newsroom.core.resources.model import NewshubResourceModel
+
+
+class SectionFilter(NewshubResourceModel):
+    name: str
+    description: Optional[str]
+    sd_product_id: Optional[str] = None
+    query: Optional[str] = None
+    is_enabled: bool = True
+    filter_type: Optional[str] = "wire"
```

Comment:
Marking `filter_type` and `search_type` as `Optional` means they accept a value of `None. These should only contain a string, i.e. non-nullable. As we're already providing a default string value here, the type for these two can simply be `str` without the `Optional`